RELEASE.md

```diff
@@ -1,39 +1,39 @@
 # Release Process
-zfs-localpv follows a monthly release cadence. The scope of the release is determined by contributor availability. The scope is published in the [Release Tracker Projects](https://github.com/orgs/openebs/projects/10).
+zfs-localpv follows a on quaterly release cadence for minor version releases. The scope of the release is determined by contributor availability. The scope is published in the [Release Tracker Projects](https://github.com/orgs/openebs/projects/78).
```

Comment:
Start with something like 'LocalPV ZFS follows or tries to follow semantic versioning principles as specified here <https://semver.org>'. 
We can do this for all main repos release.md files.

Type -> quarterly*